transcom/mymove
# B-18172 H&AL Prime Create PPM

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A871864&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

When the prime creates a PPM for the service member it should be displayed in the History and Audit Log

### How to test

1. Access the office app
2. Login as a prime
3. Open a move
4. Add a PPM Shipment
5. Logout of prime and in as a TOO/SC
6. Open the same move and navigate to the move history
7. New PPM should be listed

### Frontend
